### DISCUSSION TITLE
-export-type flag obsolete?

### DISCUSSION THREAD

**Discussion by bweisshaupt** [rusq/slackdump#137] [Q&A] [answered]
Hallo!
I downloaded slackdump-v2.1.2-windows-amd64 and ran following command:
`slackdump -base export.zip -export-type standard -download https://xxx.slack.com/archives/yyy`

Following error message was shown: 
`flag provided but not defined: -export-type`

The command `slackdump -h` does not list the -export-type option mentioned in the [user guide](https://github.com/rusq/slackdump/blob/master/doc/usage-export.rst). Is this flag deprecated?
I know that "standard" would be the default for the export type. Nevertheless clarification would help. Thank you in advance.

**Comment by rusq** [comment, marked as answer]
Hey @bweisshaupt,

Please use [2.2.0-alpha](https://github.com/rusq/slackdump/releases/tag/v2.2.0-alpha), 2.1.2 did not have this flag yet :)
